Plots/Bar/NCL_bar_3.py

```diff
@@ -0,0 +1,88 @@
+"""
+NCL_bar_3.py
+============
+This script illustrates the following concepts:
+    - Drawing bars instead of curves in an XY plot
+    - Changing the aspect ratio of a bar plot
+    - Changing the width of the bars in a bar plot
+    - Drawing filled bars up or down based on a Y reference value
+    - Setting the minimum/maximum value of the Y axis in a bar plot
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/bar_3.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/bar_3_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/soi.nc"))
+dsoik = ds.DSOI_KET
+date = ds.date
+num_months = np.shape(date)[0]
+
+# Dates in the file are represented by year and month (YYYYMM)
+# representing them fractionally will make ploting the data easier
+# This produces the same results as NCL's yyyymm_to_yyyyfrac() function
+date_frac = np.empty_like(date)
+for n in np.arange(0, num_months, 1):
+    yyyy = int(date[n] / 100)
+    mon = (date[n] / 100 - yyyy) * 100
+    date_frac[n] = yyyy + (mon - 1) / 12
+
+###############################################################################
+# Plot
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(12, 6))
+ax = plt.axes()
+
+# Create a list of colors based on the color bar values
+colors = ['red' if (value > 0) else 'blue' for value in dsoik[::8]]
+
+# Plot bar chart with defined width
+plt.bar(date_frac[::8],
+        dsoik[::8],
+        align='edge',
+        edgecolor='black',
+        color=colors,
+        width=.25,
+        linewidth=.6)
```

Comment:
```suggestion
plt.bar(date_frac[::8],
        dsoik[::8],
        color=colors,
        width=.25)
```

You don't actually need the `align='edge'` here, as that was specific to the previous example. I would also suggest removing the black outlines as it doesn't appear to be on the original I think. The lines are very small though, so feel free to disagree with me on that point.